### PR TITLE
refactor(form): use form actions instead of toolbar

### LINKED ISSUE #1289: Refactor forms to not use pf-l-toolbar for form actions
https://github.com/patternfly/patternfly-next/pull/1266/

We've removed the Toolbar layout in core, so we need to remove things that use it currently. This introduces an element in the form component called `.pf-c-form__actions` to contain the form actions instead of the Toolbar layout.

### PR DESCRIPTION
**Pull request by boaz0**
**What**:

closes #1289 

//cc @mcoker

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Form/ActionGroup.js [hunk 1]
```diff
@@ -23,13 +22,9 @@ const ActionGroup = ({ className, children, ...props }) => {
   const classesHorizontal = css(styles.formHorizontalGroup);
```
Comment (kmcfaul) [inline packages/patternfly-4/react-core/src/components/Form/ActionGroup.js]: This is unused with the removal of the isHorizontal switch.

Comment (mcoker) [reply packages/patternfly-4/react-core/src/components/Form/ActionGroup.js]: I _think_ you want to add that part back in. The actions area is broken in the horizontal form example. The `.pf-c-form__actions` element needs a `.pf-c-form__horizontal-group` parent to get the proper alignment in a horizontal form.

screen shot 2019-02-13 at 5 09 16 pm

**Comment by boaz0** [comment]
Thanks for your feedback @kmcfaul & @mcoker 
Do you mind taking a second look?

**Comment by mcoker** [comment]
@boaz0 looks like the build failed and the preview doesn't reflect the latest changes.

**Comment by boaz0** [comment]
Yeah, I think the build still has the old changes. I will try to rebase and build again and hopefully travis and circleci will pass which eventually update the build with the latest changes.

Thanks :smiley:

**Comment by boaz0** [comment]
@redallen it looks like CircleCI failed:

>#!/bin/bash -eo pipefail
yarn build:prdocs &&
cp -r packages/patternfly-4/react-docs/public .public/patternfly-4 &&
cp -r packages/patternfly-4/react-docs/public/assets .public/assets

> yarn run v1.12.3
$ lerna run pr-build
lerna notice cli v3.11.0
lerna info versioning independent
lerna info ci enabled
lerna info Executing command in 1 package: "yarn run pr-build"
lerna ERR! yarn run pr-build exited 1 in '@patternfly/react-docs'
lerna ERR! yarn run pr-build stdout:
$ node build/copyDocs.js && env NODE_OPTIONS=--max_old_space_size=4096 gatsby build --verbose --prefix-paths
>   Error: component---tmp-examples-simple-application-launcher-js-62ce2eef62b39e6  4397c.js from Terser
  undefined

**Comment by redallen** [comment]
CircleCI shouldn't even show a status check on Github for now -- I wish I could disable it. It only deals with deploying a PatternFly-React preview and isn't required to merge. If you restart the CircleCI build that specific error will go away and hopefully you'll get an updated https://1379-pr-patternfly-react-patternfly.surge.sh link.

**Comment by boaz0** [comment]
OK rebasing :crossed_fingers: :smile_cat:

**Comment by redallen** [comment]
The CircleCI build doesn't matter (in fact, I'm trying to find a way to hide it). The Travis build is fine now, he doesn't need to rebase again!

**Comment by boaz0** [comment]
Thanks @redallen !
@tlabaj @christiemolloy @mcoker sorry for bothering you again but can you please review this change again?

Thanks.

**Comment by boaz0** [comment]
Thank you all :fireworks: :tada:

**Pull request merged**